test/util_test.dart

```diff
@@ -20,6 +20,7 @@ import 'package:analyzer/dart/ast/ast.dart';
 import 'package:meta/meta.dart';
 import 'package:path/path.dart' as p;
 import 'package:pub_semver/pub_semver.dart';
+import 'package:react/react.dart';
```

Comment:
is this import necessary? I would expect it to cause issues since this is a VM test suite

Reply:
oh nope. I forgot to check for lints 🤦‍♀️